test/train-sets/ccb_test.dat

```diff
@@ -6,7 +6,7 @@ ccb action |Action ff
 ccb action |Action fff
 ccb slot 0:0:0.2 |Slot h
 ccb slot 1:0:0.25 |Slot i
-ccb slot 2:0:0.333333 |Slot j
+ccb slot 2:0:0.333333 | j
```

Comment:
Made this slot have default namespace. Doesn't affect the other test it participates in but helps test the interactions (didn't want to add another data file, but can do if pple think this is a problem)